ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/priority-fee-input/priority-fee-input.js

```diff
@@ -101,7 +107,11 @@ const PriorityFeeInput = () => {
         detailText={`≈ ${priorityFeeInFiat}`}
         numeric
       />
-      <AdvancedGasFeeInputSubtext latest="1-18 GWEI" historical="23-359 GWEI" />
+      <AdvancedGasFeeInputSubtext
+        latest="1-18 GWEI"
+        historical="23-359 GWEI"
+        feeTrend={feeTrends.priorityFeeTrend}
```

Comment:
We can avoid passing `feeTrend` and rather use it from context in `AdvancedGasFeeInputSubtext`

Reply:
Currently, we are not passing any identifier to `AdvancedGasFeeInputSubtext` to distinguish between `baseFeeTrend` and `priorityFeeTrend`, this way we don't have to make much in `AdvancedGasFeeInputSubtext`.